src/EnergyPlus/HybridModel.cc

```diff
@@ -0,0 +1,153 @@
+// ObjexxFCL Headers
+#include <ObjexxFCL/Array.functions.hh>
+
+// EnergyPlus Headers
+#include <HybridModel.hh>
+#include <DataGlobals.hh>
+#include <DataHeatBalance.hh>
+#include <DataPrecisionGlobals.hh>
+#include <DataRoomAirModel.hh>
+#include <General.hh>
+#include <HeatBalanceManager.hh>
+#include <InputProcessor.hh>
+#include <ScheduleManager.hh>
+#include <UtilityRoutines.hh>
+
+namespace EnergyPlus {
+
+namespace HybridModel {
```

Comment:
This new module only holds the get input function.  I would have expected the main hybrid model calculation function to be here as well.  Looks like the guts of the model have been added to ZoneTempPredictorCorrector::CorrectZoneAirTemp.  That was probably done because this model relies on a bunch of local variables that are present in CorrectZoneAirTemp, so one has to question if this new module is even useful or if it should be rolled into ZoneTempPredictorCorrector.  @Myoldmopar ?